src/sentry/web/frontend/debug/debug_new_processing_issues_email.py

```diff
@@ -27,10 +27,13 @@ def get(self, request):
         project = Project(
             id=1,
             organization=org,
-            team=team,
             slug='project',
             name='My Project',
         )
+        ProjectTeam(
+            project=project,
+            team=team
+        )
```

Comment:
Doesn't this just construct a `ProjectTeam` object and then throw it away? Doesn't seem like we need to do this, instead we can just remove the `team` parameter from the project no? Or does this somehow insert it into memory?

Reply:
It kept giving me an error of `teams` wasn't a valid field for `Project` for whatever reason. Perhaps it's because we aren't saving anything, instead just constructing objects.

Reply:
@EvanPurkhiser is that blocking?

Reply:
> teams wasn't a valid field for Project

Right, you can just remove this field. You don't need to add the `ProjectTeam` object. In fact, you can remove the Team object as well. It doesn't look like these debug views use teams for anything :)